medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -12,40 +15,62 @@ use tokio::time::delay_for;
 
 #[tracing::instrument]
 pub async fn delete_service_worker(
-    storage_container_name: String,
+    time_to_wait: Duration,
+    state_manager: StateManager,
 ) -> Result<String, anyhow::Error> {
+    tracing::info!("Starting delete service worker");
     let mut delete_client = delete_factory().await.map_err(|e| {
         tracing::error!("{:?}", e);
-        anyhow!("Couldn't create the delete queue client")
+        anyhow!("Couldn't delete the Delete Queue")
```

Comment:
Wouldn't this fail because it couldn't connect to the Delete queue?

Reply:
IDK what I was thinking here. Probably a search and replace fail.